### ISSUE TITLE
Decarb Washup meeting

### ISSUE THREAD

**Issue by adamj89** [Green-Software-Foundation/decarbonize-software-event#44]
- [ ] - https://github.com/Green-Software-Foundation/decarbonize-software-event/issues/40
- [ ] - https://github.com/Green-Software-Foundation/decarbonize-software-event/issues/42
- [ ] - https://github.com/Green-Software-Foundation/decarbonize-software-event/issues/41

### Technical issues

- [x] Break slides didn't trigger - do video in future
- [x] Audio issue on CODE session. `Speaker was up too loud on presenter side. Insist on rehearsal + headphones in future.` ALSO `Possible presenter central muting from production side in future`
- [x] Presentation/Screenshare issue on CODE session (mixup?) `need more strictly defined in ROS in future + rehearsal` ALSO `avoid web browser version of Teams in future or get other solution`
- [x] Video not loaded in for Champions session, playback issues (video delivered late) `break up into smaller chunks` `deliver video early` `rehearse` `mark in ROS`
- [x] Presentation issue (visibility) for Carbon Aware SDK session (not sure what happened) `rehearsals` `slides arriving earlier` `clearly advise presenters where they should be looking`

**Comment by adamj89** [comment]
Adam's top of mind

Good OKRs
Grateful for contribution of Sophie, Namrata, Matt & Alexis

**Comment by adamj89** [comment]
Sophie Top of mind:
Good event, lots of questions
How to answer those questions in a follow-up blog

**Comment by adamj89** [comment]
Namrata top of mind:
"Resounding success!"
Lots of messages about how people loved the event. Liked the content
Loved the practical guidance given along the general theory. 
AI sessions stood out. 
Chris' session really great (Adam: we will do internal version)
Lots of appreciation for CODE university session. Lots of questions about dashboards and how helpful they are .

Production standpoint: 
Smooth sailing for much of the event, a few blips.
### Speaker feedback: 
Chat was a bit noisy perhaps. (Matt has echoed this) Distracting for the speakers. It was disruptive for a few. **Can we have a backchannel chat for hosts & production only?**
Clock didn't work out. How can we communicate better when folks are nearing the end of their session.

**Comment by adamj89** [comment]
Matt feedback:
- Could we have done some as prerecord?
- Sophie: Asim would probably prefer live
- Good retention. Break caused some reduction. (It did last longer) Make the break more interesting. 
- Presentation merging issues from Google slides & different styles
- Slides need to come in earlier.

**Comment by adamj89** [comment]
Possible use of VMIX Call in the future. However slightly higher latency.

**Comment by adamj89** [comment]
Backgrounds: try and avoid awkward backgrounds / approved backgrounds in future. Preferably no custom backgrounds/blurring. Mitigation in increased rehearsals.

**Comment by adamj89** [comment]
Matt: would love to give presenters the ability to advance slides themselves using internet clicker software. Needs rehearsal time/coaching.

**Comment by Sophietn** [comment]
I had another thought - I found it pretty hard doing the Q&A. Even though we made a plan for other co-hosts to share the Qs, I still found myself mildly panicking trying to find the next Q in our teams chat and the other chat with audience, and not really listening to the speakers answer. 
Suggestion 
-ensure we get min 3 canned Qs from speakers ahead of call. 
-other co-hosts copy audience Qs to Q&A place in live script doc, that we all have open.
